python/kfserving/kfserving/protocols/seldon_http.py

```diff
@@ -0,0 +1,74 @@
+from kfserving.protocols.protocol import RequestHandler
+from http import HTTPStatus
+import tornado
+import tensorflow as tf
+from tensorflow.core.framework.tensor_pb2 import TensorProto
```

Comment:
As discussed in the meeting today, but I'll leave for posterity:

I'm willing to hear the arguments for TensorProto + HTTP, but this does make the KFServing binary much heavier than before.  It seems like proto over http is something of an antipattern. I know that Seldon aims to keep complete parity between the GRPC and REST interfaces, but at some point, it may not be worth maintaining parity.

Reply:
Removed dependency for now. When we add gRPC handling we will most likely need it anyway.